oam/install_scripts/post-install

```diff
@@ -93,53 +95,65 @@ if [ -f libstdc++.so.6.0.14 ]; then
 fi
 
 # softlink for libperl.sp, used by cplogger
-$SUDO ln -s /usr/lib64/perl5/CORE/libperl.so /usr/lib64/libperl.so >/dev/null 2>&1
-
-#setup profile files
-profileFileAlias=/etc/profile.d/columnstoreAlias.sh
-profileFileEnv=/etc/profile.d/columnstoreEnv.sh
+ln -s /usr/lib64/perl5/CORE/libperl.so /usr/lib64/libperl.so >/dev/null 2>&1
```

Comment:
Won't work as not-root, but we shouldn't do that anyway.  Ben can fix the linker options in cmake.  Unless I'm crazy.